ui/app/routes/allocations/allocation/task/fs/path.js

```diff
@@ -0,0 +1,15 @@
+import Route from '@ember/routing/route';
+
+export default Route.extend({
+  model({ path }) {
+    return {
+      path: decodeURIComponent(path),
+      task: this.modelFor('allocations.allocation.task'),
```

Comment:
Little suggestion, could you just call the `task` property `model`? Then you don't need to rename it below?

```javascript
setupController(controller, model) {
  this._super(...arguments);
  controller.setProperties(model);
}
```
It's clear from the `modelFor` argument that it's a task

Reply:
I don’t know that this quite reaches convention status, but I’ve referred to this before as a “compound model”. I haven’t seen many conventions around the naming of its components apart from them being named the things that they are. I think it would be somewhat confusing to have the `model` hook be loading something that contains a `path` and a `model`.

I hope some day there’s a more first-class way to do this kind of data loading!

Reply:
The other option would be to keep task and not rename it to `model` in `setupController`. I think my main point was more not having to rename it than deciding what to call it.

Reply:
Ideally I wouldn't have to do model hook shenanigans to pass that param to the controller, but if the param changes without the model changing then the `setupController` hook doesn't refire. 

But I also wouldn't get caught up on this. I expect this route to more or less be entirely rewritten when @backspace works on the file explorer component. Hard to say now what the best way to provide context to the controller is.

Reply:
Sweet yeah, gotcha was just a tiny thing I noticed, no biggie.